cime_config/buildexe

```diff
@@ -66,24 +83,49 @@ def _main_func():
     gmake_args += " IAC_PRESENT=FALSE"
     expect((num_esp is None) or (int(num_esp) == 1), "ESP component restricted to one instance")
 
-    with open('Filepath', 'w') as out:
-        out.write(os.path.join(caseroot, "SourceMods", "src.drv") + "\n")
-        out.write(os.path.join(cimeroot, "src", "drivers", "nuopc", "mediator") + "\n")
-        out.write(os.path.join(cimeroot, "src", "drivers", "nuopc", "drivers", "cime") + "\n")
-
-
-    # build model executable
+    bld_root = os.path.join(exeroot,'cpl','obj')
+    if not os.path.isdir(bld_root):
+        os.makedirs(bld_root)
 
+    nemssrcdir = os.path.join(srcroot, "src", "model", "NEMS", "src") 
```

Comment:
should use 
nemssrcdir = os.path.join(case.get_value(COMP_ROOT_DIR_CPL,  {"component":"nems"}),os.pardir,"src")

Reply:
this is the correct version that I use to update that part of code `nemssrcdir = os.path.join(case.get_value("COMP_ROOT_DIR_CPL"),os.pardir,"src")`